src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  // margin: 0 auto;
```

Comment:
I've read somewhere in the suggestions that we should delete commented code.

Reply:
yep, better keep commits clean